src/main.rs

```diff
@@ -251,16 +257,22 @@ mod tests {
     use axum::body::Body;
     use axum::http;
     use axum::http::{header, Request, StatusCode};
+    use camino::Utf8PathBuf;
+    use mktemp::Temp;
     use mockito::{Mock, ServerGuard};
     use reqwest::Url;
     use tower::util::ServiceExt;
 
     async fn dummy_app() -> (ServerGuard, Router) {
+        let temp_dir = Temp::new_dir().unwrap();
+        let cache_dir = Utf8PathBuf::from_path_buf(temp_dir.to_path_buf()).unwrap();
```

Comment:
does `.to_string_lossy()...` not work? Haven't seen `camino` being used before :)